website/other/bezier-rs-demos/wasm/src/bezier.rs

```diff
@@ -0,0 +1,256 @@
+use crate::svg_drawing::*;
+use bezier_rs::{ArcStrategy, ArcsOptions, Bezier, ProjectionOptions, ToSVGOptions};
+use glam::DVec2;
+use serde::{Deserialize, Serialize};
+use wasm_bindgen::prelude::*;
+
+#[derive(Serialize, Deserialize)]
+struct CircleSector {
+	center: Point,
+	radius: f64,
+	#[serde(rename = "startAngle")]
+	start_angle: f64,
+	#[serde(rename = "endAngle")]
+	end_angle: f64,
+}
+
+#[derive(Serialize, Deserialize)]
+struct Point {
+	x: f64,
+	y: f64,
+}
+
+#[wasm_bindgen]
+pub enum WasmMaximizeArcs {
+	Automatic, // 0
+	On,        // 1
+	Off,       // 2
```

Comment:
If you want to be explicit in the ordering you can assign explicit integer tag values:
```rs
enum MyEnum {
    A = 123,
    B = 456,
}
```

Reply:
The 0/1/2 comments are in reference to the possible integer values of the range slider widget to which each of these correspond, so I don't think that's necessary unless I misunderstand your intent.

Reply:
The ordering for this enum does not really matter when using the library. The ordering is only for the UI slider representation of the 3 options which is unique to the `bezier-rs-demos` web app.

Reply:
OK, I misunderstood what this was signifying.